ui/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Flask==2.2.2
 flask_cors==3.0.10
+Werkzeug==2.2.2
```

Comment:
Yeah I thought it was added before but apparently not.  It is just pinning the version since the 3.x doesn't work with flask 2.  Alternatively could probably upgrade Flask, but this was easy.